src/lib/EventPublisher.js

```diff
@@ -95,6 +97,8 @@ class EventPublisher extends EventEmitter {
           responseListener,
           {
             once: true,
+            onceUnsubscribe: true,  // Unsubscribe will be called by Eventpublisher
```

Comment:
I guess we don't need one more flag, if `once: true` we should unsubscribe after event comes no matter what.